wcomponents-core/src/test/java/com/github/bordertech/wcomponents/container/TransformXMLInterceptor_Test.java

```diff
@@ -59,26 +59,12 @@ public static void tearDownClass() {
 	}
 
 	/**
```

Comment:
You could keep this test, with setting the parameter to false and prove it still transforms.